docs/howto/how-to-cross-build-runtime-for-arm.md

```diff
@@ -1,2 +1,134 @@
 # How to Cross-build Runtime for ARM
 
+## Prepare ARM Ubuntu RootFS
+
+Install required packages
+
+```
+$ sudo apt-get install qemu qemu-user-static binfmt-support debootstrap
+```
+
+Use `build_rootfs.sh` script to prepare Root File System. You should have `sudo`
```

Comment:
```suggestion
Use `install_rootfs.sh` script to prepare Root File System. You should have `sudo`
```
I tried cross-build, but there was no file with that name.
Instead, there was a file with `install_rootfs.sh`.
I saw commit history, the file name changed, but the guide did not.

Reply:
@dev-hue , Thanks!